Neos.Flow/Tests/Unit/Security/Authorization/FilterFirewallTest.php

```diff
@@ -37,7 +37,7 @@ public function configuredFiltersAreCreatedCorrectlyUsingLegacySettingsFormat()
 
             if ($args[0] === 'URI') {
                 return 'mockPatternURI';
-            } elseif ($args[0] === 'TYPO3\\TestRequestPattern') {
+            } elseif ($args[0] === 'Neos\\TestRequestPattern') {
```

Comment:
The name of that test contains `UsingLegacySettingsFormat` - is that needed still? Also, it fails, so maybe there is a settings that needs to be adjusted (`Testing` context!?)